tscreen_unix.go

```diff
@@ -102,15 +102,19 @@ func (t *tScreen) disengage() {
 // so that it can be restored when the application terminates.
 func (t *tScreen) initialize() error {
 	var err error
-	t.out = os.Stdout
-	if t.in, err = os.Open("/dev/tty"); err != nil {
+	t.inFd, err = syscall.Open("/dev/tty", syscall.O_RDONLY|syscall.O_NONBLOCK, 0644)
```

Comment:
This should not be necessary -- I believe os.Open() will do what you need, and you can call SetNonBlocking() if need be.

Reply:
I already tried this (on Linux) to no effect.